client/src/components/posts/PostsList.js

```diff
@@ -0,0 +1,63 @@
+import { useState, useEffect } from "react";
+import { getAllPosts } from "../../managers/postManager";
+import {
+    Accordion,
+    AccordionBody,
+    AccordionHeader,
+    AccordionItem,
+  } from 'reactstrap';
+
+export default function PostsList () {
+    const [posts, setPosts] = useState([]);
+    const [open, setOpen] = useState('0');
+
+    useEffect(() => {
+        getAllPosts().then(setPosts);
+    }, []);
+    
+    const toggle = (id) => {
+        if (open === id) {
+            setOpen('0');
+        } else {
+            setOpen(id);
+        }
+    };
+
+    if (!posts) {
```

Comment:
This will never run because posts is substantiated as an empty array and empty arrays resolve to truthy. You could check for length instead.